packages/lodestar/src/chain/validation/block.ts

```diff
@@ -8,10 +8,6 @@ import {MAXIMUM_GOSSIP_CLOCK_DISPARITY} from "../../constants";
 import {IBeaconChain} from "../interface";
 import {BlockGossipError, BlockErrorCode, GossipAction} from "../errors";
 import {RegenCaller} from "../regen";
-import {byteArrayEquals} from "../../util/bytes";
-
-// TODO - TEMP: Placeholder, to be agreed with other clients
-const MAX_TRANSACTIONS_SIZE = 50e6; // 50MB
```

Comment:
Can you do an issue to add logic to reject messages over `GOSSIP_MAX_SIZE_MERGE` `MAX_CHUNK_SIZE_MERGE`? From https://github.com/ethereum/consensus-specs/blob/dev/specs/merge/p2p-interface.md#configuration

Reply:
have a few Qs regarding this GOSSIP/CHUNK size, moved  to new tracker https://github.com/ChainSafe/lodestar/issues/3425 incase we would want to go ahead and merge this

Reply:
Sounds good